config/gradle/common.gradle

```diff
@@ -16,6 +16,8 @@ apply plugin: 'com.github.spotbugs'
 apply plugin: 'jacoco'
 apply plugin: 'org.sonarqube'
 
+apply plugin: 'terasology-repositories'
```

Comment:
🎉

Reply:
the `$RESOLUTION_REPO` and `-PalternativeResolutionRepo` things I haven't tested. There's some chance of some URI type-conversion hiccup happening there, but probably not and it should be a quick fix if there is.